### PR TITLE
Current Account

### LINKED ISSUE #29: Redesign "Conta Corrente" Page

### PR DESCRIPTION
**Pull request by thePeras**
Closes #29

<div>
Screenshot 2023-04-03 at 17 28 51
Screenshot 2023-04-03 at 17 28 59
Screenshot 2023-04-03 at 17 29 07

</div>

### CODE REVIEW

File: content-scripts/src/modules/initialize.js [hunk 1]
```diff
@@ -20,3 +21,223 @@ export const injectOverrideFunctions = () => {
...
   document.body.appendChild(script);
 }
+
+/**
+ * Reverse the date direction
+ * Ex: 2023-10-05 to 05-10-2023
+ */
+export const reverseDateDirection = () => {
+  document.querySelectorAll(".data").forEach(date => {
+      date.innerHTML = date.innerHTML.split('-').reverse().join('-');
```
Comment (ttoino) [inline content-scripts/src/modules/initialize.js:31]: This may be a little dangerous, we should probably check if the content is an actual date firts

**Review by ttoino — COMMENTED** [review]
The page looks good overall, but I'm getting this error in the console and for some reason, none of the icons are getting changed 

image

File: content-scripts/src/modules/initialize.js (hunk 1 above)
Comment (thePeras) [reply content-scripts/src/modules/initialize.js:31]: Agree
What you think about convert the string to Date and verify if it creates a valid Date object?
`date instanceof Date && !isNaN(date)`

Comment (ttoino) [reply content-scripts/src/modules/initialize.js:31]: Sounds like a plan

**Comment by thePeras** [comment]
> initialize.js

I can't reproduce, but It's a good idea to checks if there are tabs

**Review by ttoino — CHANGES_REQUESTED** [review]
I'm still getting an error, it seems to be caused by me having different tabs than you do

image

**Review by ttoino — APPROVED** [review]
It seems to be working now, good job!

**Pull request merged**